### PR TITLE
feat: Added admin dropdown to new navbar

### PR DESCRIPTION
**Pull request by DedsecKnight**

### CODE REVIEW

**Review by vvyn — DISMISSED** [review]
lgtm

File: components/AppHeader2/core.tsx [hunk 1]
```diff
@@ -1,13 +1,47 @@
...
+  isCheckIn: boolean;
+  startTime: Date;
+  endTime: Date;
+  isPermanentScan: boolean;
+};
 
+export default function AppHeader2_Core() {
+  const { user, hasProfile } = useAuthContext();
+  const router = useRouter();
+  const isSuperAdmin = user.permissions.indexOf('super_admin') !== -1;
```
Comment (uy-seng) [inline components/AppHeader2/core.tsx]: sometimes i get a user with type of null, so maybe we can do user?.permissions...

File: components/AppHeader2/core.tsx [hunk 2]
```diff
@@ -1,13 +1,47 @@
...
+  const [scanList, setScanList] = useState<Scan[]>([]);
+  const [currentScan, setCurrentScan] = useState<Scan | null>(null);
+  useEffect(() => {
+    if (!isAdmin) {
+      setScanList([]);
+    }
+    async function getScanData() {
+      const scans = await RequestHelper.get<Scan[]>('/api/scantypes', {
+        headers: {
+          authorization: user.token || '',
```
Comment (uy-seng) [inline components/AppHeader2/core.tsx]: same for this part here

File: components/AppHeader2/core.tsx [hunk 3]
```diff
@@ -1,13 +1,47 @@
...
+  const isSuperAdmin = user.permissions.indexOf('super_admin') !== -1;
+  const isAdmin = isSuperAdmin || user.permissions.indexOf('admin') !== -1;
+  const [scanList, setScanList] = useState<Scan[]>([]);
+  const [currentScan, setCurrentScan] = useState<Scan | null>(null);
+  useEffect(() => {
+    if (!isAdmin) {
+      setScanList([]);
+    }
+    async function getScanData() {
+      const scans = await RequestHelper.get<Scan[]>('/api/scantypes', {
```
Comment (uy-seng) [inline components/AppHeader2/core.tsx:32]: api request return 401 (unauthorized), shouldn't this just return nothing if its normal user?

File: components/AppHeader2/core.tsx [hunk 4]
```diff
@@ -1,13 +1,47 @@
...
+        headers: {
+          authorization: user.token || '',
+        },
+      });
+      setScanList(scans.data);
```
Comment (uy-seng) [inline components/AppHeader2/core.tsx:37]: maybe its better to do try...catch or .then and set the scanlist just in case scans return some error

**Review by uy-seng — APPROVED** [review]
lgtm

**Review by vvyn — APPROVED** [review]

**Pull request merged**